lib/jekyll-haml/version.rb

```diff
@@ -1,5 +1,5 @@
 module Jekyll
   module Haml
-    VERSION = '0.1.6'.freeze
+    VERSION = '2.0.0'.freeze
```

Comment:
This version doesn't match the changeling, but otherwise happy to merge.

Reply:
Oops.  Fixed that.  Should be OK now.

Reply:
Thank you. Appreciate the update.

Reply:
@mfaughn I forgot to push the new version until just now. Hope that works well for you.
https://rubygems.org/gems/jekyll-haml/versions/1.0.0

Reply:
OK by me.  FWIW, the powers that be prevent me from using any Jekyll plugins at all anymore.  Hopefully this changes in the future...